test/TestListRemote.re

```diff
@@ -0,0 +1,138 @@
+open TestFramework;
+
+describe("List Remote", ({test}) => {
+  let allVersions6 = [
```

Comment:
what do you think about moving this into the test scope itself?

Reply:
Done :)